### PR TITLE
Support Page e2e

### PR DESCRIPTION
**Pull request by etanb**
I've re-enabled the Support page e2e test and it now covers:

- Loading of page with correct title, headers etc
- That the primary Contact Support link is visible and matches what's in production
- Dynamically goes through the 3 primary information cards for accuracy
- Scrolls to proper section on clicking "View all topics"

### CODE REVIEW

**Comment by penny-lischer** [comment]
@etanb we should implement the POM here. So updating the support.ts file to extend the BasePage.

File: frontend-react/e2e/spec/all/support-page.spec.ts [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
         await support.goto(page);
     });
 
-    test("should have correct title", async ({ page }) => {
-        await expect(page).toHaveURL(/support/);
-        await expect(page).toHaveTitle(/ReportStream support/);
+    test("Should have the correct page header", async ({ page }) => {
+        const header = page;
+
+        await await expect(header).toHaveText("h1", "Support");
```
Comment (jpandersen87) [inline frontend-react/e2e/spec/all/support-page.spec.ts:29]: Linting is angry at the `toHaveText` method here.

**Review by jpandersen87 — CHANGES_REQUESTED** [review]
Needs to be POM style, and there's a linting issue.

File: frontend-react/e2e/spec/all/support-page.spec.ts [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
 
-    test("should have correct title", async ({ page }) => {
-        await expect(page).toHaveURL(/support/);
-        await expect(page).toHaveTitle(/ReportStream support/);
+test.describe("Support page", () => {
+    test("Should have a way of contacting support", async ({ supportPage }) => {
+        // Importing the actual URL of our contact form to match
+        // what we use in production and only need to make updates
+        // in a single place
+        const jsonPath = path.resolve("./src/content/site.json");
```
Comment (jpandersen87) [inline frontend-react/e2e/spec/all/support-page.spec.ts:59]: You can use the modern way of importing a json file in node: `import site from "../../../src/content/site.json" assert { type: "json" };`

**Review by jpandersen87 — APPROVED** [review]

File: frontend-react/e2e/spec/all/support-page.spec.ts (hunk 2 above)
Comment (etanb) [reply frontend-react/e2e/spec/all/support-page.spec.ts]: didn't seem to work on the first go around for some reason....

**Review by penny-lischer — APPROVED** [review]
LGTM

**Pull request merged**